test/e2e/framework/context_create_operations.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package framework
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+// ContextCreateOps helps to run context create command
+type ContextCreateOps interface {
+	// CreateConextWithEndPoint creates a context with a given endpoint URL
+	CreateConextWithEndPoint(contextName, endpoint string) error
+	// CreateConextWithEndPoint creates a context with a given endpoint URL
+	CreateConextWithEndPointStaging(contextName, endpoint string) error
```

Comment:
Just for understanding: Why do we need `--staging` when creating context with `--endpoint`?

Reply:
its not working without --staging option.

❯ tz context create --name tmc13 --endpoint unstable.tmc-dev.cloud.vmware.com            
✖  Failed to obtain access token. Please provide valid VMware Cloud Services API-token -- {"metadata":null,"errorCode":null,"statusCode":400,"requestId":"2ac7e0fcb9594e5c","message":"invalid_grant: Invalid refresh token: xxxx...cbamo","cspErrorCode":"540.120-340.800","moduleCode":540,"traceId":"95c9b01ad3455ddd2dd558a366a6cbbe"} 
❯ tz context create --name tmc13 --endpoint unstable.tmc-dev.cloud.vmware.com --staging

✔  successfully created a TMC context 
ℹ  Checking for required plugins... 
!  unable to automatically sync the plugins from target context. Please run 'tanzu plugin sync' command to sync plugins manually 
❯